CRM/Eck/BAO/Entity.php

```diff
@@ -41,4 +44,23 @@ public static function getEntityIcon(string $entityName, int $entityId):?string
     return $record['subtype:icon'] ?? $default;
   }
 
+  /**
+   * Implements hook_civicrm_post().
+   *
+   * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_post
+   */
+  public static function on_hook_civicrm_post(GenericHookEvent $event) {
```

Comment:
Same as above: Return type could be `void` and `$event` could be type hinted more specific.